### PR TITLE
feat: style project resources component

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR adds full styling for the project resources UI widget, adding icons, links, and sorting out the layout.

Light mode:
image

Dark mode:
image

On narrower screens:
image

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx [hunk 1]
```diff
@@ -31,14 +87,56 @@ export const ProjectResources = () => {
...
-                <h3>Project Resources</h3>
-                <span>{project.members} project member(s)</span>
-                <span>{tokens.length} API key(s)</span>
-                <span>1 SDK connection(s)</span>
-                <span>{segmentCount} project segment(s)</span>
+                <Typography variant='h3' sx={{ margin: 0 }}>
+                    Project Resources
+                </Typography>
+                <ResourceList>
+                    {makeListItem(
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx:112]: I think more reach way would be if this is react component, which takes icon as child and all other as props. 

<ListItem text={{}} ....><UsersIcon /></ListItem>

Comment (sjaanus) [reply frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx:112]: `<ListItem text={{}} ><UserIcon /></ListItem>`

**Review by sjaanus — APPROVED** [review]

File: frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx (hunk 1 above)
Comment (thomasheartman) [reply frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx:112]: Yeah, that could work. I experimented a bit with it, but didn't quite find what I wanted, but lemme have another look.

Comment (thomasheartman) [reply frontend/src/component/project/Project/ProjectStatus/ProjectResources.tsx]: Updated in 8a325ec94bdccad059de2c705e0f21d17b633b98.

**Pull request merged**